### PR TITLE
Reza_framework_Framework_section

### PR DESCRIPTION
**Pull request by reza-github**

### CODE REVIEW

**Comment by dhruvdhody** [comment]
Hi Reza, 

In the figure added here why do you feel it is needed to show multiple Transport Network Controllers? Maybe you think of these as per-domain controllers? But the TSC SBI examples are L3SM, L2SM, ACTN CMI etc which are service models usually implemented at the orchestrator / MDSC level (see the figure in section 'Mapping to ACTN' for my understanding)! 

Thoughts?

**Comment by reza-github** [comment]
Hi Dhruv,

In general I agree with you that in most cases there is one TSC which
interact via its NBI to the "higher system" and via its SBI to one or
multiple Domain controller to manages all transport slices.
However, there might be some use-cases where having multiple TSC MIGHT
needed. An example is 5G where a single TSC can handle and manages the
transport slices on
Fronthaul, Midhaul and Backhaul. However, it might be possible to have
multiple TSCs for each of these.

Again having multiple TSC is just future looking to make sure our framework
is inclusive of any new use-cases.
In summary, I agree that in most cases there should be one TSC for entire
system to manages all transport slices.
We should discuss this during the call today.

Reza

**Comment by reza-github** [comment]
> Hi Reza,
> 
> In the figure added here why do you feel it is needed to show multiple Transport Network Controllers? Maybe you think of these as per-domain controllers? But the TSC SBI examples are L3SM, L2SM, ACTN CMI etc which are service models usually implemented at the orchestrator / MDSC level (see the figure in section 'Mapping to ACTN' for my understanding)!
> 
> Thoughts?

Hi Dhruv,
You are right for some solutions to realization of the transport slice on TSC SBI. For example as you indicated using L2SM, L3SM etc needs only one Transport network controller. 
However, in some cases for realization of transport slices, it is possible that the TSC needs to talk to various domains directly. 
My intention was to allow one or more transport network controller to make sure we cover all cases.

**Comment by dhruvdhody** [comment]
Hi Reza,

Yes, there could be multiple Transport Network Controller if there are multiple administrative domains which each being served by a different controller. To match this with the merged section ( https://github.com/teas-wg/teas-ns-dt/blob/master/framework/draft-framework.md#mapping-to-actn ) i feel that it would be clear if we show a single Transport Network Controller and you can mention it in words that in case of multiple administrative domains, there could be different controllers serving them. 

In your proposed figure, this gives an impression of domain controllers (per AS, IGP area domain controllers - basically a PNC) which i wanted to avoid, as it would be confusing for TEAS readers.

Thanks! 
Dhruv

**Comment by reza-github** [comment]
> Hi Reza,
> 
> Yes, there could be multiple Transport Network Controller if there are multiple administrative domains which each being served by a different controller. To match this with the merged section ( https://github.com/teas-wg/teas-ns-dt/blob/master/framework/draft-framework.md#mapping-to-actn ) i feel that it would be clear if we show a single Transport Network Controller and you can mention it in words that in case of multiple administrative domains, there could be different controllers serving them.
> 
> In your proposed figure, this gives an impression of domain controllers (per AS, IGP area domain controllers - basically a PNC) which i wanted to avoid, as it would be confusing for TEAS readers.
> 
> Thanks!
> Dhruv

Hi Dhruv,
I think we have to state the multiplicity of the "Transport Network Controllers" both in picture and text. If you think the picture is not clear, what your suggestion is to show multiplicity in picture and the same time remove the mis-interpretation.
Reza
